promql/parse.go

```diff
@@ -29,9 +29,8 @@ import (
 )
 
 type parser struct {
-	lex     *Lexer
-	token   Item
-	peeking bool
```

Comment:
Due to the parser rewrite `peeking` is always `false` now. So it can be as well removed.